src/core/specter-wallet.ts

```diff
@@ -267,7 +267,7 @@ export class SpecterWallet {
       throw new Error(err)
     }
 
-    const tx = await this.bitcoindClient.getTransaction({
+    const tx = await bitcoindDefaultClient.getTransaction({
```

Comment:
Can we please avoid coupling to global state / objects. No singletons... use constructor injection.